src/app/modules/create-facility/pages/facility-info/facility-info.component.html

```diff
@@ -1,8 +1,169 @@
+          </common-error-container> -->
+        </div>
+
+        <div class="form-group">
+          <common-province label='Province' value='' formControlName="mailing_province" disabled required="true">
+          </common-province>
+          <!-- <common-error-container [displayError]="validFormControl(mailingForm,'mailing_province')">
+            Province is a required field
+          </common-error-container> -->
+        </div>
+
+        <div class="form-group">
+          <common-postal-code label='Postal Code' value='' formControlName="mailing_postalCode" class="pt-1"
+            required="true">
+          </common-postal-code>
+          <!-- <common-error-container [displayError]="validFormControl(mailingForm,'mailing_postalCode')">
+            Postal Code is a required field
+          </common-error-container> -->
+        </div>
+
+      </form>
+    </div>
+
+
+    <h2 class='border-bottom'>Facility Service Type</h2>
+    <div class="form-group col-sm-8 p-sm-0">
+      <form [formGroup]="facilityForm">
+        <div formArrayName="services" *ngFor="let fg of getServiceTypes.controls; let formIndex=index;">
+          <div class="form-group">
+            <common-checkbox class="form-check mb-3"
+              formControlName="{{availableServiceTypes[formIndex].value}}"
+              id="{{availableServiceTypes[formIndex].value}}" label="{{availableServiceTypes[formIndex].label}}"
+              required="true">
+            </common-checkbox>
+          </div>
+          <common-error-container [displayError]="validFormControl(facilityForm,'services')">
+            At least one (1) Service Type must be selected
+          </common-error-container>
+        </div>
+      </form>
+    </div>
+
+
+    <h2>Business Cost Premium</h2>
+    <p class="border-bottom">
+      Business Cost Premium applies to services which are provided in community-based offices where the physician pays
+      towards rent or lease cost either directly or indirectly.
+    </p>
+    <div class="form-group col-sm-8 p-sm-0">
+      <form [formGroup]="facilityForm">
+        <div class="form-group">
+          <common-radio [radioLabels]="radioBtnLabels" formControlName="isQualifyForBCP"
+            label="Does your business qualify for the Business Cost Premium?" display="table-row-group" required="true">
+          </common-radio>
+          <common-error-container [displayError]="validFormControl(facilityForm,'isQualifyForBCP')">
+            Please indicate if your business qualifies for the Business Cost Premium
+          </common-error-container>
+        </div>
+      </form>
+    </div>
+
+  </common-page-section>
 </common-page-framework>
 
-<common-form-action-bar [defaultColor]='false'
-                        widthOption='extra-width-mobile-only'
-                        (btnClick)="continue()">
-</common-form-action-bar>
+<common-form-action-bar [defaultColor]='false' widthOption='extra-width-mobile-only' [isLoading]='loading'
```

Comment:
You don' need to do `[isLoading]='loading'` here since you'll never show a spinner on this page I believe.  No harm leaving i in though, so I'll merge the review :)